### PR TITLE
ZENKO-2878 test ui elements

### PR DESCRIPTION
**Pull request by romainlancia**

### CODE REVIEW

File: src/react/ui-elements/__tests__/Clipboard.test.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        const { component } = reduxMount(<Clipboard text={OWNER_NAME}/>);
+
+        expect(component.find(Clipboard)).toHaveLength(1);
+        expect(component.find('#clipboard-icon-copy')).toHaveLength(1);
+    });
+
+    it('Clipboard should render with success icon', () => {
+        const writeTextFn = jest.fn();
+
+        global.navigator.clipboard = { writeText: writeTextFn };
```
Comment (nicolas2bert) [inline src/react/ui-elements/__tests__/Clipboard.test.jsx:17]: Do we need to clean the clipboard afterwards?

File: src/react/ui-elements/__tests__/DeleteConfirmation.test.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+        expect(component.find(DeleteConfirmation).isEmptyRender()).toBe(true);
+    });
+
+    it('should call approve function after clicking on delete button', () => {
+        const { component } = reduxMount(<DeleteConfirmation approve={approveFn} cancel={cancelFn} show={true}
+            titleText={TITLE_TEXT}/>);
+
+        expect(component.find(DeleteConfirmation).isEmptyRender()).toBe(false);
+        component.find('Button[text="Delete"]').simulate('click');
```
Comment (nicolas2bert) [inline src/react/ui-elements/__tests__/DeleteConfirmation.test.jsx:30]: 👍

File: src/react/ui-elements/__tests__/Activity.test.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        expect(component.find('#activity-message').children().at(2).text()).toBe(TEST_MESSAGE);
+    });
+
+    it('Activity should render with a default message', () => {
+        const { component } = reduxMount(<Activity/>, {
+            networkActivity: { counter: 2, messages: List() },
+        });
+
+        expect(component.find(Activity).isEmptyRender()).toBe(false);
+        expect(component.find('#activity-message').children().at(2).text()).toBe('Working...');
```
Comment (nicolas2bert) [inline src/react/ui-elements/__tests__/Activity.test.jsx]: Can you please export/import the default message?

Comment (romainlancia) [reply src/react/ui-elements/__tests__/Activity.test.jsx]: done

File: src/react/ui-elements/__tests__/Clipboard.test.jsx (hunk 1 above)
Comment (romainlancia) [reply src/react/ui-elements/__tests__/Clipboard.test.jsx:17]: I don't think it is necessary

File: src/react/ui-elements/Activity.jsx [hunk 4]
```diff
@@ -5,6 +5,8 @@ import React from 'react';
 import styled from 'styled-components';
 import { useSelector } from 'react-redux';
 
+export const ACTIVITY_TEXT_WORKING = 'Working...';
```
Comment (nicolas2bert) [inline src/react/ui-elements/Activity.jsx]:
```suggestion
export const DEFAULT_MESSAGE = 'Working...';
```

**Review by nicolas2bert — APPROVED** [review]

File: src/react/ui-elements/Activity.jsx (hunk 4 above)
Comment (romainlancia) [reply src/react/ui-elements/Activity.jsx]: done

**Review by nicolas2bert — APPROVED** [review]

**Review by anurag4DSB — APPROVED** [review]

**Review by terrydervaux — COMMENTED** [review]
@lebroz / @nicolas2bert  Is there a test for this modal?
Screenshot 2020-10-22 at 11 42 37

**Comment by nicolas2bert** [comment]
Yes @terrydervaux, this one: src/react/ui-elements/__tests__/ErrorHandlerModal.test.jsx

**Comment by romainlancia** [comment]
@bert-e approve

**Pull request merged**